hmpps-prisoner-search-indexer/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/repository/PrisonerRepository.kt

```diff
@@ -171,10 +171,10 @@ class PrisonerRepository(
       client.lowLevelClient.performRequest(request)
       return true
     } catch (e: ResponseException) {
-      log.error("Unexpected response ${e.response} from delete of $prisonerNumber", e)
       if (e.response.statusLine.statusCode == 404) {
         return false
       }
+      log.error("Unexpected response ${e.response} from delete of $prisonerNumber", e)
```

Comment:
small tweak avoiding an incorrect exception log when the prisoner is just not in the index